server/sonar-web/src/main/js/apps/component-measures/components/MeasureContentContainer.js

```diff
@@ -112,18 +112,18 @@ export default class MeasureContentContainer extends React.PureComponent {
     );
   };
 
-  updateLoading = (loading: { [string]: boolean }) => {
+  updateLoading = (loading /*: { [string]: boolean } */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)